composer.json

```diff
@@ -14,13 +14,24 @@
     }
   ],
   "autoload": {
-    "psr-0": {
-      "PAMI": "src/mg"
+    "psr-4": {
+      "PAMI\\": "src/mg/PAMI"
     }
   },
   "require": {
     "php": ">=5.3.3",
     "apache/log4php": "2.3.0",
     "psr/log": "~1.0"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "4.8.*",
```

Comment:
That's interesting. I took that one from: https://phpunit.de/manual/current/en/installation.html